js/foundation.dropdownMenu.js

```diff
@@ -154,13 +154,13 @@ class DropdownMenu extends Plugin {
             hasSub = $elem.hasClass(parClass);
         if (hasSub && _this.options.autoclose) {
           if ($elem.attr('data-is-click') === 'true' && _this.options.clickOpen) { return false; }
-          // Check for hovering over input dropdown options
-          if(e.relatedTarget === null && document.hasFocus && document.hasFocus()) {
-            return false;
-          }
 
           clearTimeout($elem.data('_delay'));
           $elem.data('_delay', setTimeout(function() {
+            // Check for hovering over input dropdown options
```

Comment:
@ncoden It seems placing this check in the `setTimeout` rather than where I had it before resolves the issue of the dropdown staying open on Firefox when the user uses the keyboard to navigate to another window. The only issue I've found of it still staying open on Firefox with this change is if the user is already hovered over the input suggestions and then uses the keyboard to navigate to another window. But I wouldn't consider that a breaking change, since currently, the dropdown disappears if the user is hovered over one of those suggestions. I'm going to update the PR accordingly.